<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>268</width>
    <height>193</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Run</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <widget class="QLineEdit" name="MaxIterationTime">
   <property name="geometry">
    <rect>
     <x>132</x>
     <y>132</y>
     <width>125</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>99</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LoadStep">
   <property name="geometry">
    <rect>
     <x>132</x>
     <y>72</y>
     <width>125</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>12</y>
     <width>93</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Analysis type:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ErrorTolerance">
   <property name="geometry">
    <rect>
     <x>132</x>
     <y>102</y>
     <width>125</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0.001</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>102</y>
     <width>97</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Error Tolerance:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>132</y>
     <width>123</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Max iteration time:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LoadFactor">
   <property name="geometry">
    <rect>
     <x>132</x>
     <y>42</y>
     <width>125</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>1.0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>42</y>
     <width>77</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Factor:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>72</y>
     <width>77</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Step:</string>
   </property>
  </widget>
  <widget class="QToolButton" name="OKButton">
   <property name="geometry">
    <rect>
     <x>86</x>
     <y>162</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QToolButton" name="CancelButton">
   <property name="geometry">
    <rect>
     <x>176</x>
     <y>162</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>96</x>
     <y>12</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Static linear analysis</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Static nonlinear analysis</string>
    </property>
   </item>
  </widget>
 </widget>
 <tabstops>
  <tabstop>comboBox</tabstop>
  <tabstop>LoadFactor</tabstop>
  <tabstop>LoadStep</tabstop>
  <tabstop>ErrorTolerance</tabstop>
  <tabstop>MaxIterationTime</tabstop>
  <tabstop>OKButton</tabstop>
  <tabstop>CancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
